apps/ui/src/networks/types.ts

```diff
@@ -63,12 +63,18 @@ export type StrategyConfig = StrategyTemplate & {
   params: Record<string, any>;
 };
 
+export type SnapshotInfo = {
+  at: number | null;
+  chain_id?: number;
```

Comment:
```suggestion
  chainId: number;
```

Can we rename it to use camelCase as it's what we tend to use. Space/Proposals use kebab_case but it's mostly due to the API shape (which we should solve in the future), everything else uses camelCase.